grow/pods/document_fields.py

```diff
@@ -0,0 +1,80 @@
+"""
+Document fields for accessing the meta fields parsed from the document.
+"""
+
+from boltons import iterutils
+import re
+
+
+LOCALIZED_KEY_REGEX = re.compile('(.*)@([^@]+)$')
+
+
+class DocumentFields(object):
```

Comment:
This is great, and a long time coming. :) I wonder if we should just call it "Fields"? To keep things simple...